fe/src/main/java/org/apache/doris/load/loadv2/BrokerLoadJob.java

```diff
@@ -112,27 +109,31 @@ public static BrokerLoadJob fromLoadStmt(LoadStmt stmt, String originStmt) throw
         if (db == null) {
             throw new DdlException("Database[" + dbName + "] does not exist");
         }
-        // check data source info
-        LoadJob.checkDataSourceInfo(db, stmt.getDataDescriptions(), EtlJobType.BROKER);
 
         // create job
         try {
             BrokerLoadJob brokerLoadJob = new BrokerLoadJob(db.getId(), stmt.getLabel().getLabelName(),
-                                                            stmt.getBrokerDesc(), stmt.getDataDescriptions(),
-                                                            originStmt);
+                    stmt.getBrokerDesc(), originStmt);
             brokerLoadJob.setJobProperties(stmt.getProperties());
-            brokerLoadJob.setDataSourceInfo(db, stmt.getDataDescriptions());
+            brokerLoadJob.checkAndDataSourceInfo(db, stmt.getDataDescriptions());
```

Comment:
`checkAnd` ?? Maybe `checkAndCreate` is better.

Reply:
change to `checkAndSetDataSourceInfo`